Tasks/Npm/npm.ts

```diff
@@ -30,6 +33,88 @@ async function main(): Promise<void> {
     }
 }
 
+function logStartupVariables() {
```

Comment:
This method should be extracted into a common location so it can be shared between nuget and npm.

Reply:
Agree.  Will do this in a different PR.